examples/whisper/prepare_whisper_configs.py

```diff
@@ -153,7 +153,7 @@ def main(raw_args=None):
     # set model name in prepost
     template_json["passes"]["prepost"]["tool_command_args"]["model_name"] = model_name
 
-    for device, precision in SUPPORTED_WORKFLOWS:
+    for device, precision in SUPPORTED_WORKFLOWS.items():
```

Comment:
this needs to be `(device, precision), workflow in SUPPORTED_WORKFLOW.items()`. The keys are tuples so the old code was technically correct.

Reply:
i see. will revert this back. Thanks!

Reply:
sounds good! or if you prefer you could combine lines 156-157 with the new .items() change and no ignores are needed.